examples/models/phi-3-mini/export_phi-3-mini.py

```diff
@@ -48,8 +47,14 @@ def main() -> None:
         model(*example_inputs)
         model = convert_pt2e(model, fold_quantize=False)
         DuplicateDynamicQuantChainPass()(model)
-        model = export(
-            model, example_inputs, dynamic_shapes=dynamic_shape, strict=False
+        # TODO(lunwenh): update it to use export once
+        # https://github.com/pytorch/pytorch/issues/128394 is resolved.
+        model = torch.export._trace._export(
```

Comment:
Why this though? can we still do torch.export.export()?

Reply:
`export` currently fails due to https://github.com/pytorch/pytorch/issues/128394. We will use this trick until that issue is fixed.